pkgs/top-level/ruby-packages.nix

```diff
@@ -3929,10 +4326,10 @@
     platforms = [ ];
     source = {
       remotes = [ "https://rubygems.org" ];
-      sha256 = "0b6v4d5y042qd9grsfr2hh82i3lvc7lkjlf0jab1037dy10926nv";
+      sha256 = "0i1h1kr5j6v8kzrcdfrjxdf1qnnrwhxl9i5vkr8wgbspp5raissq";
       type = "gem";
     };
-    version = "1.4.0";
+    version = "1.3.0";
```

Comment:
Intentional downgrade?

Reply:
cant find this in new diff anymore

Reply:
I don't understand, it's right next to this comment.

Reply:
not in Gemfile